### PR TITLE
Major refactor of tutorial folder structure.

### PR DESCRIPTION
**Pull request by morrieinmaas**
* In accordance with isseue 61 description on GitHub

TODOs:

* Fix all broken links within the notebooks
* extend notebooks to cover mediation and multitenant
* possibly introduce helper functions that cn be imported from extra
folder
* !!! If medation and multitenant live in folder 1. the
docker-compose.yaml and manage script need to be adjusted to accomodate
for that setup. TBD what that looks like. More info
[here](https://github.com/hyperledger/aries-cloudagent-python/blob/main/Multitenancy.md#multi-tenant-admin-api)

## Description
see [issue 61](https://github.com/OpenMined/PyDentity/issues/61)

## Checklist
- [ ] I have followed the [Contribution Guidelines](https://github.com/OpenMined/.github/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/OpenMined/.github/blob/master/CODE_OF_CONDUCT.md)
- [ ] I have commented my code following the [OpenMined Styleguide](https://github.com/OpenMined/.github/blob/master/STYLEGUIDE.md)
- [ ] I have labeled this PR with the relevant [Type labels](https://github.com/OpenMined/.github/labels?q=Type%3A)
- [ ] My changes are covered by tests

### CODE REVIEW

**Comment by wip-abramson** [comment]
Sorry to be a pain but wondering about a slight refactor of tutorial structure for the first tutorial.

Basically Proofs tutorials wont work unless they have first issued themselves a credential. So perhaps it makes sense just to condense into the Credentials folder?

I also think Out of Band tutorial is an Advanced concept not a messaging one.

Like the tutorial names though!

**Comment by morrieinmaas** [comment]
@wip-abramson I'll refactor like that. Makes sense to me.

File: configuration/aries-args-advanced.yaml [hunk 1]
```diff
@@ -57,3 +57,4 @@ debug-connections: true
 debug-credentials: true
 debug-presentations: true
 enable-undelivered-queue: true
+open-mediation: true
```
Comment (wip-abramson) [inline configuration/aries-args-advanced.yaml:60]: We should specify this in the docker-compose instead

File: libs/aries-basic-controller/aries_basic_controller/aries_controller.py [hunk 2]
```diff
@@ -35,10 +37,13 @@ def __init__(
         webhook_base: str = "",
         connections: bool = True,
         messaging: bool = True,
+        multitenant: bool = False,
+        mediation: bool = True,
         issuer: bool = True,
         action_menu: bool = True,
         revocations: bool = True,
         api_key: str = None,
+        jwt_token: str = None,
```
Comment (wip-abramson) [inline libs/aries-basic-controller/aries_basic_controller/aries_controller.py:46]: can we call this subwallet_token or something like that

File: libs/aries-basic-controller/aries_basic_controller/aries_controller.py [hunk 3]
```diff
@@ -77,12 +94,23 @@ def __init__(
...
             self.action_menu = ActionMenuController(self.admin_url, self.client_session)
+
         if revocations:
             self.revocations = RevocationController(
                 self.admin_url,
                 self.client_session
             )
 
+        
+    # def update_jwt_token(self, api_key: str):
```
Comment (wip-abramson) [inline libs/aries-basic-controller/aries_basic_controller/aries_controller.py:105]: I think we should have this.
something like set_subwallet_token

**Comment by morrieinmaas** [comment]
@wip-abramson @lohanspies Please have a look. I'm happy for any feedback. Lesson learned, pulled too much into one task. that got a bit out of hand volume-wise.

**Comment by lohanspies** [comment]
> at got a bit out of hand volume-wise.

Agree. Let us keep it focussed on small achievable chunks.

File: configuration/aries-args-multitenant.yaml [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+auto-respond-credential-request: false
+auto-store-credential: false
+auto-respond-presentation-proposal: true
+auto-respond-presentation-request: true
+preserve-exchange-records: true
+debug-connections: true
+debug-credentials: true
+debug-presentations: true
+enable-undelivered-queue: true
+open-mediation: true
```
Comment (wip-abramson) [inline configuration/aries-args-multitenant.yaml:60]: We should move this out of here

**Review by wip-abramson — APPROVED** [review]
All looks good to me, good work

**Pull request merged**